client/ngapp/index.html

```diff
@@ -41,30 +46,30 @@
     <div ng-view></div>
 
     <!--[if lt IE 9]>
-    <script src="bower_components/es5-shim/es5-shim.js"></script>
-    <script src="bower_components/json3/lib/json3.min.js"></script>
+    <script src="vendor/es5-shim/es5-shim.js"></script>
+    <script src="vendor/json3/lib/json3.min.js"></script>
     <![endif]-->
 
-    <!-- build:js(.) scripts/vendor.js -->
+    <!-- build:js(client/ng-app) scripts/vendor.js -->
     <!-- bower:js -->
-    <script src="../bower_components/es5-shim/es5-shim.js"></script>
-    <script src="../bower_components/angular/angular.js"></script>
-    <script src="../bower_components/json3/lib/json3.js"></script>
-    <script src="../bower_components/angular-route/angular-route.js"></script>
+    <script src="../../bower_components/es5-shim/es5-shim.js"></script>
+    <script src="../../bower_components/angular/angular.js"></script>
+    <script src="../../bower_components/json3/lib/json3.js"></script>
+    <script src="../../bower_components/angular-route/angular-route.js"></script>
```

Comment:
This looks suspiciously to me. Based on your comments above, I would expect these paths should reference `vendor`, not `bower_components`? The `if lt IE 9` block uses `vendor` paths, which seems to be inconsistent with the paths here.

Thoughts?